CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/select/BoxSelectionPlugin.java

```diff
@@ -58,7 +58,8 @@ public BoxSelectionPlugin(final boolean isAdd, final boolean isToggle, final Cam
     @Override
     public void edit(final GraphWriteMethods graph, final PluginInteraction interaction, final PluginParameters parameters) throws InterruptedException {
 
-        final float mix = camera.getMixRatio();
+        //Divide by 20 to change the mix range from 0-20 to 0-1
+        final float mix = camera.getMixRatio() / 20.0f;
```

Comment:
it looks like camera provides a method for doing this in `Camera#getMix()` - this would avoid the magic number.

Reply:
Good find. Didn't even see that. Made the change